tests/test_jsonwrite.py

```diff
@@ -0,0 +1,60 @@
+import temporalimage
+import unittest
+import jsonschema
+from jsonschema import validate
```

Comment:
this import is breaking the circleCI validation. you have two options: (1) remove jsonschema; or (2) include jsonschema in requirements.txt and setup.py

Reply:
I like the use of jsonschema; it would be good to include it as one of the requirements and also use this kind of validation in other places in the code